androidApp/src/main/res/values/strings.xml

```diff
@@ -198,16 +198,14 @@
     <string name="onboarding_feedback_advance">Get started</string>
     <string name="onboarding_feedback_body">MBTA Go is in the early stages! We want your feedback as we continue making improvements and adding new features.</string>
     <string name="onboarding_feedback_header">Help us improve</string>
-    <string name="onboarding_hide_maps_body">When using TalkBack, we can skip reading out maps to keep you focused on transit information.</string>
-    <string name="onboarding_hide_maps_header">Set map preference</string>
-    <string name="onboarding_hide_maps_hide">Hide maps</string>
-    <string name="onboarding_hide_maps_show">Show maps</string>
     <string name="onboarding_location_advance">Continue</string>
```

Comment:
If we want all the “Continue”s to be the same, we should get rid of this one, too.